config.json

```diff
@@ -681,7 +681,8 @@
       "unlocked_by": null,
       "difficulty": 1,
       "topics": [
-
+        "lists",
```

Comment:
I like your choice. In #900 additional topics were suggested:
```text
        "variables",
        "lists",
        "loops",
        "filtering",
        "searching",
        "conditionals",
        "callbacks"
```
Could you please add them as well?